clone.js

```diff
@@ -1,55 +1,61 @@
-import fetch from 'node-fetch';
+// Get your free API Key at: https://www.docubee.com/solutions/integrations/docubee-api
+// Full Docubee API Documentation: https://docs.docubee.app/#overview
+
+const docubeeUrl = 'https://docubee.app/api/v2';
```

Comment:
This should be discussed with the team. As this is sample code for our users, the URL should not change or need to be parameterized. While I would normally write a wrapper around an API in this way, it may be less straightforward for the end user to not see the full URL of the call in the actual code.